### ISSUE TITLE
on Win10 

### ISSUE THREAD

**Issue by zxj11838** [xingyizhou/CenterNet#843]
Is it necessary to use Visual Studio when used under Win10?

**Comment by xingyizhou** [comment]
I don't think you need Visual Studio. See [here](https://github.com/xingyizhou/CenterNet/issues/7) for a reference for running on Windows.
